lib/webdrivers/system.rb

```diff
@@ -26,7 +26,7 @@ def delete(file)
       end
 
       def install_dir
-        Webdrivers.install_dir || File.expand_path(File.join(ENV['HOME'], '.webdrivers'))
+        Webdrivers.install_dir
```

Comment:
is this different from just using the `attr_accessor`?

Reply:
The accessor (reader method) was only returning the user define path or `nil`. I attempted to move all the logic in `System.install_dir` and have `Webdrivers.install_dir` reference it, but then that created a circular reference as the first check in System is  for `Webdrivers.install_dir`, which itself was referencing `System.install_dir` to get the default path.